src/blocks/stars.css

```
@@ -0,0 +1,18 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  margin-right: 4px;
+
+  height: 16px;
+  width: 16px;
+
+  background-image: url(../images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars__star--active {
+  background-image: url(../images/star-active.svg);
```

this one part we have to change accordingly html modifier. but you still able to look for the beter way.

(check the PATH'es to files )

```suggestion
.stars--1 .stars__star:first-child,
.stars--2 .stars__star:nth-child(-n+2),
.stars--3 .stars__star:nth-child(-n+3),
.stars--4 .stars__star:nth-child(-n+4),
.stars--5 .stars__star:nth-child(-n+5) {
  background-image: url(../images/star-active.svg);
```

---

first i did it through html and after i understood that beter to do this from css